src/main/java/com/bouncestorage/swiftproxy/v1/AccountResource.java

```diff
@@ -179,6 +184,29 @@ public BulkDeleteResult bulkDelete(@NotNull @PathParam("account") String account
             }
         }
 
+        removeBlobsMap.forEach((container, blobList) ->  {
+            try {
+                blobStore.removeBlobs(container, blobList);
+                result.numberDeleted += blobList.size();
+            } catch (ContainerNotFoundException e) {
+                result.numberNotFound += blobList.size();
+            } catch (Exception e) {
+                e.printStackTrace();
+                blobList.forEach(blob -> result.errors.add(container + "/" + blob));
+            }
+        });
+        deleteContainers.forEach(container -> {
+            try {
+                blobStore.deleteContainer(container);
+                result.numberDeleted += 1;
+            } catch (ContainerNotFoundException e) {
+                result.numberNotFound += 1;
+            } catch (Exception e) {
+                e.printStackTrace();
```

Comment:
in this case I disagree, we can't propagate the exception so unless we print it, it will not be possible to find out why something wrong happened